pkgs/tools/admin/meshcentral/yarn.nix

```diff
@@ -1,60 +1,148 @@
-{ fetchurl, fetchgit, linkFarm, runCommand, gnutar }: rec {
+{ fetchurl, fetchgit, linkFarm, runCommandNoCC, gnutar }: rec {
```

Comment:
You should use `yarn2nix` from `master`, this doesn't evaluate anymore because `runCommandNoCC` is deprecated and thus the ofborg eval-check fails.

Yes, it's painful and I really dislike this decision, but that's where we are :)

Reply:
Fixed, thanks!